wizeline/project-lab
# #108 - Make sure only team members can edit a proposal and other peop…

#### What does this PR do?

PBI 108 - Validate that only the team members can modify or delete a proposal.

#### Where should the reviewer start?
Proposals Module.

#### How should this be manually tested?
- Create a proposal (to become owner) try to update and delete it, the expected result is that you can do those operations.
- Try to edit or delete an existing proposal, where you're not a team member, the expected result would be you can't update or delete the proposal.
- Try to edit or delete an exiting proposal, where you're a team member, the expected result would be you can do the operation successfully.

#### Any background context you want to provide?
Before this feature, anyone could modify any of the proposals.

#### What are the relevant tickets?
https://github.com/wizeline/project-lab/issues/108

Screenshots
When you're a team member
![image](https://user-images.githubusercontent.com/44985743/141593368-5f363cbe-184d-4adc-9671-21ac1d073dba.png)

When you're not a team member
![image](https://user-images.githubusercontent.com/44985743/141593388-cb23ab67-bf30-438a-aac4-6f7af3930912.png)

#### Checklist
- [ X] I added the necessary documentation, if appropriate.
- [ X] I reviewed existing Pull Requests before submitting mine.
